src/dags/importscripts/import_processenverbaalverkiezingen.py

```diff
@@ -0,0 +1,134 @@
+import csv
+import ftplib
+from more_ds.network.url import URL
+from os import path as ospath
+from typing import List, TypedDict, Tuple, Optional, Generator
+
+
+class Data(TypedDict):
+    """Datastructure processenverbaal verkiezingen"""
+
+    verkiezingsjaar: int
+    volgnummer: str
+    uri: str
+    documentnaam: str
+    stemlocatie: str
+    id: str
+
+
+class FTPListing:
+    """This class is used for traversing a given folder base on a FTP
+    server (objectstore) using BFS (Breadth-First search) algorithm.
+    """
+
+    def __init__(self, connection: ftplib.FTP) -> None:
+        """Initialize
+
+        Args:
+            Connection: Hold the connection to the objectstore
+
+        """
+        self.connection = connection
+
+    def list_dirs_and_files(self, _path: str) -> Tuple[List[str], Optional[List[str]]]:
```

Comment:
```python
-> Tuple[List[str], List[str]]
```
The second element of the returned tuple (`files`) will never be `None`. In the body of the function you initialize it to `[]` and you are only appending to it. Hence `files` will never be `None`

Reply:
Having said that you might want to consider using a dataclass. See https://github.com/Amsterdam/dataservices-airflow/pull/203/files#r585502118

Reply:
:+1: